inz-internetnz/PrivateKeys/1.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
If `1.xml` is present then `0.xml` is never used, even though `0.xml` is also valid. Recommend removing this (and also the corresponding `1.xml` in the `PublicKeys` folder.